### DISCUSSION TITLE
Welcome to Sprout Discussions!

### DISCUSSION THREAD

**Discussion by AIGamesDeveloper** [Sprout-Inc/sprout-website#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Report any Bugs
  * Add/Remove words from the whitelist
  * Ask questions
  * Have fun

**Comment by Official-Soogle** [comment]
I cant download sprout

**Reply by AIGamesDeveloper** [reply]
> I cant download sprout

Please tell me the following info
OS:
Decription:
